ArcGISRuntimeSDKQt_CppSamples/Search/OfflineGeocode/OfflineGeocode.cpp

```diff
@@ -314,19 +314,22 @@ void OfflineGeocode::onGeocodingCompleted_(const QList<GeocodeResult>& geocodeRe
   m_geocodeInProgress = false;
   emit geocodeInProgressChanged();
 
-  // if there are no matching results, notify user
-  if (geocodeResults.empty())
+  // dismiss callouts
+  m_calloutData->setVisible(false);
+  m_pinGraphic->setVisible(false);
+
+  // if there are no matching results, notify user and stop processing
+  if (geocodeResults.isEmpty())
```

Comment:
I changed to `isEmpty` purely for aesthetics. It is the same API and played no part in the problems.

https://doc.qt.io/qt-6/qlist.html#empty